ReactiveCocoa/Swift/FoundationExtensions.swift

```diff
@@ -37,12 +37,12 @@ extension NSNotificationCenter {
 
 private let defaultSessionError = NSError(domain: "org.reactivecocoa.ReactiveCocoa.rac_dataWithRequest", code: 1, userInfo: nil)
 
-extension NSURLSession {
+extension URLSession {
 	/// Returns a producer that will execute the given request once for each
 	/// invocation of start().
-	public func rac_dataWithRequest(request: NSURLRequest) -> SignalProducer<(NSData, NSURLResponse), NSError> {
+	public func rac_dataWithRequest(_ request: URLRequest) -> SignalProducer<(Data, URLResponse), NSError> {
```

Comment:
This should be `rac_data(with request: URLRequest) -> SignalProducer(Data, URLResponse), NSError>`